src/main/scala/cromwell/Main.scala

```diff
@@ -42,7 +42,11 @@ object Main extends App {
    * Also now passing args to runAction instead of the constructor, as even sbt seemed to have issues with the args
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
-  val AskTimeout = Timeout(5 seconds)
+  // The magic maximum value for an ask timeout.  This value should be large to represent the amount of time until
+  // the future will fail (not the amount of time to wait for the Future to be returned from the ask).  This Future
+  // completes after the workflow goes to a terminal status, so the timeout must be at least as long as the workflow
+  // run duration.
+  val AskTimeout = Timeout(21474835 seconds)
```

Comment:
Maybe this is a sign that ask isn't the right thing to be doing? This implies you could never have a workflow which takes that long (I realize that having an ~8 month workflow running via command line would be a bit insane)